### PR TITLE
HTML/CSS template basics

### LINKED ISSUE #28: Incorporate cwd_framework_lite into starter kit
We should use the CSS Framework as the base for HTML/CSS/JS in the Starter Kit.

The [cwd_framework_lite](https://github.com/CU-CommunityApps/cwd_framework_lite) is maintained as a platform-agnostic version of the Custom Dev CSS Framework. We should create a basic Cornell branded layout and establish how we will continue to update the Starter Kit with CSS Framework revisions.

### PR DESCRIPTION
**Pull request by woodseowl**
Incorporates cwd_framework_lite assets into the install process.

- Added cwd_framework_lite to packagist via https://github.com/CU-CommunityApps/cwd_framework_lite/pull/1 
- Uses composer to get cubear/cwd_framework_lite
- Installs views/components and css/js assets
- Adds instructions for functional testing the starter kit during development
- Closes #28

**To Review:**

- [ ] Follow the Usage instructions in the updated [README.md](https://github.com/CU-CommunityApps/CD-LaravelStarterKit/blob/3f19c185b0bf41aaf229092db5808ca46440dffd/README.md#usage)
  - NOTE: In your Laravel project you need to composer require the branch for this PR rather than just the package:
  ```
     composer require cornell-custom-dev/laravel-starter-kit:dev-html-css-basics
  ```
- [ ] In your Laravel project, replace the default `resources/views/welcome.blade.php` with the newly installed `resources/views/cwd-framework-index.blade.php` and confirm that a page similar to https://iws-preview.hosting.cornell.edu/ama39/cssf/ is shown when you visit your site locally
- [ ] Read the new "Functional testing" section of [CONTRIBUTING.md](https://github.com/CU-CommunityApps/CD-LaravelStarterKit/blob/3f19c185b0bf41aaf229092db5808ca46440dffd/CONTRIBUTING.md#functional-testing) to see if it is clear enough without being overly verbose.
- [ ] Extra credit: Follow those Functional testing instructions and confirm that you can locally make changes to your Starter Kit repo and see them updated in your Laravel project without having to composer update.

Note: The way the HTML is sliced in `resources/views/components/cwd-framework` is based on a few different examples from our projects. That said, it is just a first pass and I expect it to have future revisions based on conversations with the designers about how best to split it into easily managed parts. I created an issue (#34) to make sure we address that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,24 +15,15 @@ A Cornell University CIT Custom Development starter kit package for Laravel.
...
-       "type": "vcs",
-       "url": "git@github.com:CU-CommunityApps/CD-LaravelStarterKit.git"
-     }
-   ],
-   ```
-3. Composer require the CD-LaravelStarterKit
+2. Composer require the LaravelStarterKit
    ```shell
-   composer require cu-communityapps/cd-laravelstarterkit
+   composer require cornell-custom-dev/laravel-starter-kit
```
Comment (inaydich) [inline README.md:20]: I am no sure I understand the instruction. I pulled the code from the  `html-css-basics` branch. The instaruction says to run `composer require cornell-custom-dev/laravel-starter-kit:dev-html-css-basics`
that gives an error: `Root package 'cornell-custom-dev/laravel-starter-kit' cannot require itself in its composer.json`

Comment (inaydich) [reply README.md:20]: Oh! It looks like this is for usage - for a final laravel project. Does it mean we have to add css framework as a separate package and we are not going to include it into the starter kit?

Comment (woodseowl) [reply README.md:20]: Yes, this is for how to pull in the starter kit into a new Laravel project. The css framework is brought in as part of `composer require cornell-custom-dev/laravel-starter-kit`, and the assets are installed with `artisan starterkit:install`. I think your questions point out that I need to clarify things more in the differences between the usage instructions of the README and the development / testing instructions of CONTRIBUTING. I will work on that.

File: src/StarterKitServiceProvider.php [hunk 2]
```diff
@@ -11,28 +11,53 @@
...
         '.lando.yml',
     ];
 
+    public const ASSET_FILES = [
+        'css',
+        'fonts',
+        'images',
+        'js',
+        'favicon.ico',
+    ];
```
Comment (woodseowl) [inline src/StarterKitServiceProvider.php:29]: @inaydich I think your critique of not including the sass directory was on target. I've limited the published assets to only those used by the layout.

**Review by inaydich — APPROVED** [review]
looks good

**Pull request merged**